.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: required
+language: node_js
+node_js:
+  - "8"
+
+before_install:
+  - "curl -L http://git.io/ejPSng | /bin/sh"
+
+env:
+  - CXX=g++-4.8
```

Comment:
It seems these two env lines run a pipeline each. https://travis-ci.org/apinf/apinf-fiware/builds/267003602?utm_source=github_status&utm_medium=notification We don't need travis to run twice.

Reply:
Fixed